src/main/java/bio/terra/workspace/service/resource/controlled/AccessScopeType.java

```diff
@@ -0,0 +1,47 @@
+package bio.terra.workspace.service.resource.controlled;
+
+import bio.terra.workspace.generated.model.ControlledResourceCommonFields;
+import bio.terra.workspace.service.workspace.exceptions.InternalLogicException;
+import org.apache.commons.lang3.SerializationException;
+import org.apache.commons.lang3.StringUtils;
+
+/** Controlled resources can be shared by all users in a workspace or be private to a single user */
+public enum AccessScopeType {
+    ACCESS_SCOPE_SHARED("ACCESS_SHARED"),
```

Comment:
I don't know that we need the `ACCESS_` prefix, since that's going to be the same for the whole enum.